### PR TITLE
add new author

### PR DESCRIPTION
**Pull request by plzdoo**
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/yiming.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Yiming
+short_name:
```
Comment (joshuarrrr) [inline _authors/yiming.markdown:3]: remove if not using:

```suggestion
```

**Review by joshuarrrr — CHANGES_REQUESTED** [review]
@plzdoo Please move the new file to the `_authors/osci_2023` directory.

File: _authors/yiming.markdown (hunk 1 above)
Comment (plzdoo) [reply _authors/yiming.markdown]: Thanks for suggestion, change already

**Review by dtaivpp — APPROVED** [review]
Approved!

**Pull request merged**